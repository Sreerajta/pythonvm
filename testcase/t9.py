print "calling one function inside another function"

def fun1():
 print 5+1
 
def fun2():
 print "8"
 fun1()

fun2()
