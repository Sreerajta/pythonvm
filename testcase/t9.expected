calling one function inside another function
8
6
